### PR TITLE
Update show note of episode 10

### PR DESCRIPTION
**Pull request by pofat**
Add comments from twitter and summary of ownership.

### CODE REVIEW

File: _posts/2019-09-30-10.md [hunk 1]
```diff
@@ -55,11 +54,43 @@ categories:
 * 13 的 Apple [#ArcadeReview](https://twitter.com/search?q=%23ArcadeReview%20%40ethanhuang13) 連載中
 * iPad mini 超適合玩 Apple Arcade 的，又輕又快又平💸
 
-## 歡樂的時光總是過得特別快
+## 本集播出後的後續討論與內容更正
+
+本集內容播出後，有推友向我們回饋有些內容觀念混淆也提出了正確和清楚的說法，非常感謝這些推友們，我們非常樂見這樣的~~抓漏~~討論！這些討論和更正整理如下：
+
+### Swift 的 closure 與 ObjectiveC 的 block
```
Comment (ethanhuang13) [inline _posts/2019-09-30-10.md]: Objective-C

File: _posts/2019-09-30-10.md [hunk 2]
```diff
@@ -55,11 +54,43 @@ categories:
...
+    * Dangling pointer
+    * Free after use
+    * Strong reference (retain) cycle
+* Ownership 在 Swift 的實作其實就是體現在 ARC 裡的 strong reference，這裡就是我們熟悉的，當 retain count 變成 0，instance 就會被銷毀，這裡是實作在 Swift 語言本身的機制，也是我們本集的主要內容。
+* Ownership [manifesto](https://github.com/apple/swift/blob/master/docs/OwnershipManifesto.md) 提案內容是在一般開發者無感的底層實作（ARC）之外，提供讓開發者以加標籤的方式（shared、moveOnly 等）以使用 ownership 的功能，這些功能最主要都圍繞在獨佔性原則（Law of Exclusivity）
+* 承上，Swift 裡主要想解決的問題在 copy 的性能問題，主要是針對 struct 的 copy on write
+* 照 manifesto 裡的內容，若要運用勢必會對現行的程式碼帶來改變，但他們有一再強調改變會盡量小且必要（but who knows）
+* 而且應該是要和 ARC 共存，並不是取代（這是我個人讀完的見解）
+* 就我所知目前應該尚未有可用的實作
+* 參考資料：[推友原文](https://twitter.com/kemchenj/status/1178909418261319680?s=20)，一篇分析 ownership 在 Rust 與 Swift 差異的[論文](https://macsphere.mcmaster.ca/handle/11375/23625)
```
Comment (ethanhuang13) [inline _posts/2019-09-30-10.md]: 要不要加上喵神的翻譯？

Comment (ethanhuang13) [reply _posts/2019-09-30-10.md]: https://onevcat.com/2017/02/ownership/

**Review by ethanhuang13 — CHANGES_REQUESTED** [review]

File: _posts/2019-09-30-10.md (hunk 2 above)
Comment (pofat) [reply _posts/2019-09-30-10.md]: Added in 76fdf087a457b34dcc0d757011949fd3d832c02e

File: _posts/2019-09-30-10.md (hunk 1 above)
Comment (pofat) [reply _posts/2019-09-30-10.md]: Fixed in 76fdf087a457b34dcc0d757011949fd3d832c02e

**Pull request merged**